_docs-2/configuration/server-properties.md

```diff
@@ -6,7 +6,7 @@ order: 4
 
 <!-- WARNING: Do not edit this file. It is a generated file that is copied from Accumulo build (from core/target/generated-docs) -->
 
-Below are properties set in `accumulo.properties` or the Accumulo shell that configure Accumulo servers (i.e tablet server, manager, etc). Properties labeled 'Experimental' should not be considered stable and have a higher risk of changing in the future.
+Below are properties set in `accumulo.properties` or the Accumulo shell that configure Accumulo servers (i.e. tablet server, manager, etc). Properties labeled 'Experimental' should not be considered stable and have a higher risk of changing in the future.
```

Comment:
This change is fine to keep. I'll do a quick patch to update the code that generates this, so it doesn't get clobbered on next release.

Reply:
Addressed in https://github.com/apache/accumulo/commit/74e7f3dd590d17f4ebc6a5f2b5e351b45342c578